content/tutorials/content/single-page-tutorials/docusaurus.config.js

```diff
@@ -22,7 +22,77 @@ module.exports = {
         preview: 'IOTA-Smart-Contract-Tutorials-A.jpg',
         route: 'smart-contracts/guide/chains_and_nodes/running-a-node',
         tags: ['text', 'video'],
-      },
+      }
+    ],
+    [
+      '@iota-wiki/plugin-tutorial',
+      {
+        title: 'Run a Wasp Chain',
+        description: 'In this tutorial you will learn how to run a wasp chain.',
+        preview: 'IOTA-Smart-Contract-Tutorials-B.jpg',
+        route: 'smart-contracts/guide/chains_and_nodes/setting-up-a-chain',
+        tags: ['text', 'video'],
+      }
+    ],
+    [
+      '@iota-wiki/plugin-tutorial',
+      {
+        title: 'Deploy a WASM Smart Contract',
+        description: 'In this tutorial you will learn how to deploy a wasm smart contract.',
+        preview: 'IOTA-Smart-Contract-Tutorials-C.jpg',
+        route: 'smart-contracts/guide/chains_and_nodes/setting-up-a-chain#deploying-a-wasm-contract',
+        tags: ['text', 'video'],
+      }
+    ],
+    [
+      '@iota-wiki/plugin-tutorial',
+      {
+        title: 'Run a EVM Chain',
+        description: 'In this tutorial you will learn how to run an EVM node.',
+        preview: 'IOTA-Smart-Contract-Tutorials-D.jpg',
+        route: 'smart-contracts/guide/evm/create-chain',
+        tags: ['text', 'video'],
+      }
+    ],
+    [
+      '@iota-wiki/plugin-tutorial',
+      {
+        title: 'Deploy Solidity Smart Contract using Remix + Metamask',
+        description: 'In this tutorial you will learn how to deploy a solidity smart contract on EVM using remix and metamask.',
+        preview: 'IOTA-Smart-Contract-Tutorials-E.jpg',
+        route: 'smart-contracts/guide/evm/tooling#remix',
+        tags: ['text', 'video'],
+      }
+    ],
+    [
+      '@iota-wiki/plugin-tutorial',
+      {
+        title: 'Deploy Solidity Smart Contract using Hardhat',
+        description: 'In this tutorial you will learn how to deploy a solidity smart contract on EVM using hardhat.',
+        preview: 'IOTA-Smart-Contract-Tutorials-F.jpg',
+        route: 'smart-contracts/guide/evm/tooling#hardhat',
+        tags: ['text', 'video'],
+      }
+    ],
+    [
+      '@iota-wiki/plugin-tutorial',
+      {
+        title: 'Generate WASM Smart Contract using Schema Tool',
+        description: 'In this tutorial you will learn how to use schema tool to generate smart contract template code on rust and golang',
+        preview: 'IOTA-Smart-Contract-Tutorials-G.jpg',
+        route: 'smart-contracts/guide/schema/usage',
+        tags: ['text', 'video'],
+      }
+    ],
+    [
+      '@iota-wiki/plugin-tutorial',
+      {
+        title: 'Generate WASM Smart Contract using Schema Tool in Typescript',
+        description: 'In this tutorial you will learn how to use schema tool to generate smart contract template code on typescript',
+        preview: 'IOTA-Smart-Contract-Tutorials-G.jpg',
+        route: 'https://www.youtube.com/watch?v=P5HMgmY3DMM',
```

Comment:
Absolute routes do not work out of the box. I need to add a bit of logic to support this. I will work on it tomorrow, because I agree this is a needed feature.